### ISSUE TITLE
sun set times are off by a three  minutes(compared to NOAA website and other libraries).

### ISSUE THREAD

**Issue by mb12** [kevinboone/solunar_cmdline#3]
Steps to reproduce:

1. Run solunar like this. 
./solunar -s -l 37.33085,-121.880779 -d "31/01/2018"

It shows the following:
                       Sunrise: 07:11
                        Sunset: 17:28

the sunset time should be 17:31, which is what the NOAA website shows.

https://www.esrl.noaa.gov/gmd/grad/solcalc/

Additional Info
---------------
1.) It looks like a bug because it seems to be always off by 3 minutes.

2.) I also tried the following utility. Its sunset time always matches the NOAA one. 

https://sourceforge.net/projects/solunar/?source=typ_redirect

**Comment by kevinboone** [comment]
Hi

I just fixed a bug that was causing sunset times to be 2-3 minutes too
early. The cases I tried were now within a minute of figures published by
the BBC, which is what I test against. Sorry about that -- slip of the
fingers. I just pushed the changes to the github repository.

I don't think it's realistic to expect sunrise/set times to agree to closer
than a minute or so. A difference of that magnitude could be accounted for
by differences in rounding, for example. At the same time, however, I am
aware of the fact that I am using algorithms and data that were published
in the 70s. If the results of my code are consistently different by more
than a minute or two from other reliable sources, I will have to consider
upgrading the math. However, that will be a pain, and I don't want to do it
unless I have to.

Best wishes
Kevin
